Packs/PrismaCloud/Integrations/RedLock/RedLock.yml

```diff
@@ -248,6 +264,21 @@ script:
     arguments:
     - name: alert-id
       description: comma separated list of string IDs to be dismissed
+    - name: dismissal-value
+      required: false
+      description: The amount of time to dismiss for if snoozing. Both value and unit must be specified
+    - name: dismissal-unit
+      required: false
+      auto: PREDEFINED
+      predefined:
+      - hour
+      - day
+      - week
+      - month
+      - year
+      - login
+      - epoch
+      description: The time unit for dismissial if snoozing alert.  Both dimissal value and unit must be specified if snoozing.
```

Comment:
you've added 2 arguments to the command but did not support them in the code
remove or implement in the command

Reply:
yes forgot to push the snooze code, updated now to include the code as well for these parameters.

Reply:
I still don't see support in code for this args.

Reply:
these args are used in the dismiss command lines 324-337.  
```
    snooze_value = demisto.args()['snooze-value'] if 'snooze-value' in demisto.args() else None
    snooze_unit = demisto.args()['snooze-unit'] if 'snooze-unit' in demisto.args() else None

    msg_notes = ['dismissed', 'Dismissal']

    if snooze_value and snooze_unit:
        payload['dismissalTimeRange'] = {
            'type': 'relative',
            'value': {
                'unit': snooze_unit,
                'amount': int(snooze_value)
            }
        }
        msg_notes = ['snoozed', 'Snooze']
```